accounts/views.py

```diff
@@ -0,0 +1,53 @@
+from django.shortcuts import render
+from django.contrib.auth.forms import UserCreationForm
+
+from django.contrib.auth import authenticate, login, logout
+
+from .forms import CreateUserForm
+from django.shortcuts import redirect
+
+from django.contrib import messages
+
+# Create your views here.
+
+
+def home(request):
+    return render(request, "dashboard.html")
+
+
+def registerPage(request):
+    form = CreateUserForm()
+
+    if request.method == "POST":
+        form = CreateUserForm(request.POST)
+        if form.is_valid():
+            form.save()
+            user = form.cleaned_data.get("username")
+            messages.success(request, "Account was created for user " + user)
```

Comment:
Używawj formatted stringów zamiast konkatenacji

Reply:
Zmieniłem.

Reply:
Chodziło mi o f"Account was created for user {user}"